pkg/search/postgres/store.go

```diff
@@ -192,3 +193,35 @@ func (s *GenericStore[T, PT]) GetAll(ctx context.Context) ([]*T, error) {
 	})
 	return objs, err
 }
+
+// Get returns the object, if it exists from the store.
+func (s *GenericStore[T, PT]) Get(ctx context.Context, id string) (PT, bool, error) {
+	defer s.setPostgresOperationDurationTime(time.Now(), ops.Get)
+
+	var sacQueryFilter *v1.Query
+	if s.hasPermissionsChecker() {
+		if ok, err := s.permissionChecker.GetAllowed(ctx); err != nil {
+			return nil, false, err
+		} else if !ok {
+			return nil, false, sac.ErrResourceAccessDenied
```

Comment:
This changes the behaviour compared to the generated code.
I'd stick to the previous behaviour that returns `nil, false, nil` otherwise, this may leak information about the user allowed scope.